ui/client/components/graph/node-modal/Input.js

```diff
@@ -13,12 +12,13 @@ const Input = (props) => {
                 <input
                     key={label}
                     type="text"
-                    className="node-input"
+                    className={validators.some(validator => validator.isValid(value) === false) ?
```

Comment:
Do you can create some method at validators kind of isValid / hasErrors with checks each validator.isValid?

Reply:
Done